README.md

```diff
@@ -1,3 +1,77 @@
 # tern.gee
 
-Create TLGs using Generalized Estimating Equations (GEE)
+[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
```

Comment:
maybe consider using "work in progress" status instead of "active"?

Reply:
Done!